### PR TITLE
Add Systemd service file

### PR DESCRIPTION
**Pull request by Rjasuja**
Adding service file to support systemd initialization manager

Issue: #35
Test: Verified service on ubuntu setup

Signed-off-by: Ritul Jasuja <ritul.jasuja@intel.com>

### CODE REVIEW

**Comment by Rjasuja** [comment]
@lucasdemarchi Please check after merging your diff

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,6 @@ before_script:
         - CC="gcc-6"
 
 script:
-        - ./autogen.sh c
+        - ./autogen.sh c --with-systemdsystemunitdir=/usr/lib/systemd/system
```
Comment (edersondisouza) [inline .travis.yml]: Why `pkg-config` can't get this on travis?

File: README.md [hunk 2]
```diff
@@ -39,7 +46,6 @@ Install:
     $ make install
     $ # or... to another root directory:
     $ make DESTDIR=/tmp/root/dir install
-
```
Comment (edersondisouza) [inline README.md]: Why was this line removed?

**Comment by Rjasuja** [comment]
Cancelling this because of new PR

File: .travis.yml (hunk 1 above)
Comment (lucasdemarchi) [inline .travis.yml]: bug in autoge.sh not passing the arguments correctly. Fixed in master.
